doc/md/declarative/declarative-sql.mdx

```diff
@@ -0,0 +1,176 @@
+
+CREATE TABLE orders (
+    id int NOT NULL AUTO_INCREMENT, 
+    customer_id int NOT NULL, 
+    order_date date NOT NULL, 
+    PRIMARY KEY (id), 
+    CONSTRAINT customer_id_fk FOREIGN KEY (customer_id) REFERENCES example.customers (id)
+);
+```
+
+## Step 3: Applying the change
+
+The final step is to apply the change to your database. To do this, simply run the `atlas schema apply` command. This command will compare the current state of your database to the desired state defined in your SQL file and make the necessary changes.
+
+### Usage:
+
+```console
+atlas schema apply [flags]
+```
+
+### Example:
+
+Let’s apply the changes we created in Step 2 using the following command:
+
+```console
+atlas schema apply \
+  --url "mysql://root:pass@localhost:3306/example" \
+  --to "file://schema.sql" \
+  --dev-url "docker://mysql/8/example"
+```
+
+```console
+-- Planned Changes:
+-- Create "orders" table
+CREATE TABLE `orders` (`id` int NOT NULL AUTO_INCREMENT, `customer_id` int NOT NULL, `order_date` date NOT NULL, PRIMARY KEY (`id`), INDEX `customer_id_fk` (`customer_id`), CONSTRAINT `customer_id_fk` FOREIGN KEY (`customer_id`) REFERENCES `customers` (`id`) ON UPDATE NO ACTION ON DELETE NO ACTION) CHARSET utf8mb4 COLLATE utf8mb4_0900_ai_ci;
+Use the arrow keys to navigate: ↓ ↑ → ← 
+? Are you sure?: 
+  ▸ Apply
+    Abort
+```
+
+Selecting `Apply` will apply the changes to the database.
+
+You can inspect the latest state of the database again and verify that the changes have been applied by re-running the `atlas schema inspect` command:
+
+```console
+atlas schema inspect -u "mysql://root:pass@localhost:3306/example" --log "{{ sql . }}"
+```
+
+```sql
+-- create "customers" table
+CREATE TABLE `customers` (
+    `id` int NOT NULL AUTO_INCREMENT, 
+    `first_name` varchar(255) NOT NULL, 
+    `last_name` varchar(255) NOT NULL, 
+    PRIMARY KEY (`id`)) CHARSET utf8mb4 COLLATE utf8mb4_0900_ai_ci;
+
+-- create "orders" table
+CREATE TABLE `orders` (
+    `id` int NOT NULL AUTO_INCREMENT, 
+    `customer_id` int NOT NULL, 
+    `order_date` date NOT NULL, PRIMARY KEY (`id`), 
+    INDEX `customer_id_fk` (`customer_id`), 
+    CONSTRAINT `customer_id_fk` FOREIGN KEY (`customer_id`) REFERENCES `customers` (`id`) ON UPDATE NO ACTION ON DELETE NO ACTION) CHARSET utf8mb4 COLLATE utf8mb4_0900_ai_ci;
+```
+Great! Our changes have been successfully applied to the `example` schema.
+
+### Conclusion
+
+By following the steps above, you can inspect the current state of the database, make changes to the schema definition file and apply those changes to the database with confidence.
+
+The declarative migration is a flexible way to manage your database schema. It allows you to define your desired state in a simple, human-readable language, and then automatically apply those changes to your database. This can save you time and effort, and also reduce the risk of errors or inconsistencies in your database schema.
```

Comment:
```suggestion
The declarative migration is a flexible way to manage your database schema. It allows you to define your desired state, and then automatically apply those changes to your database. This can save you time and effort, and also reduce the risk of errors or inconsistencies in your database schema.
```

It's not simple/human-readable anymore because it's not HCL